mlflow/store/artifact/runs_artifact_repo.py

```diff
@@ -134,7 +140,7 @@ def _get_logged_model_artifact_repo(
         """
         from mlflow.store.artifact.artifact_repository_registry import get_artifact_repository
 
-        client = mlflow.tracking.MlflowClient()
+        client = mlflow.tracking.MlflowClient(self.tracking_uri)
         experiment_id = client.get_run(run_id).info.experiment_id
 
         def iter_models() -> Iterator[LoggedModel]:
```

Comment:
Do we need to pass self.tracking_uri to line 161?

Reply:
yes, thanks for the catch!